src/pages/settings/Wallet/VerifyAccountPage.tsx

```diff
@@ -39,31 +37,26 @@ function VerifyAccountPage({route}: VerifyAccountPageProps) {
         User.clearContactMethodErrors(contactMethod, 'validateLogin');
     }, [contactMethod]);
 
+    const closeModal = useCallback(() => {
+        // Disable modal visibility so the navigation is animated
+        setIsValidateCodeActionModalVisible(false);
+        Navigation.goBack();
```

Comment:
We need to add backto param in case users refresh page and go back (ref: https://github.com/Expensify/App/issues/53253)